pkg/util/arp.go

```diff
@@ -199,17 +199,45 @@ func ArpDetectIPConflict(nic, ip string, mac net.HardwareAddr) (net.HardwareAddr
 	// Announcement is identical to the ARP Probe described above,
 	// except that now the sender and target IP addresses are both
 	// set to the host's newly selected IPv4 address.
-	if pkt, err = arp.NewPacket(arp.OperationRequest, mac, tpa, tha, tpa); err != nil {
+	err = BroadcastFreeArp(nic, ip, mac, announceNum, announceInterval)
+	if err != nil {
 		return nil, err
 	}
 
+	return nil, nil
+}
+
+func BroadcastFreeArp(nic, ip string, mac net.HardwareAddr, announceNum int, announceInterval time.Duration) error {
+	klog.Infof("broadcast free arp with nic %s , ip %s, with mac %v ", nic, ip, mac)
+	ifi, err := net.InterfaceByName(nic)
```

Comment:
ifi should be replaced by more meaningful name

Reply:
replaced

Reply:
fixed